Documentation/installation/cli-download.rst

```diff
@@ -10,7 +10,8 @@ various features (e.g. clustermesh, Hubble).
       CILIUM_CLI_VERSION=$(curl -s https://raw.githubusercontent.com/cilium/cilium-cli/master/stable.txt)
       CLI_ARCH=amd64
       if [ "$(uname -m)" = "aarch64" ]; then CLI_ARCH=arm64; fi
-      curl -L --fail --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${CILIUM_CLI_VERSION}/cilium-linux-${CLI_ARCH}.tar.gz{,.sha256sum}
+      URL=https://github.com/cilium/cilium-cli/releases/download/${CILIUM_CLI_VERSION}/cilium-linux-${CLI_ARCH}.tar.gz{,.sha256sum}
+      curl -L --fail --remote-name-all ${URL}
```

Comment:
I'm not a fan of the additional line here (we tried to reduce the number of lines here in the past). Tell me please, how do these escapes work, would you be able to copy-paster better if we had, say, double quotes around the URL or is it not related at all? How does the `URL` variable improve things (and why is there no change needed for the `shasum`/`tar`/`rm` lines)?